src/api-clients/document-api-client.js

```diff
@@ -122,7 +122,7 @@ class DocumentApiClient {
       .then(res => res.data);
   }
 
-  hardDeleteDocument(documentId) {
+  hardDeletePrivateDocument(documentId) {
```

Comment:
I wanted to make this clear because I kept thinking it's a leftover from when public external documents could be deleted.
Also cause of the restriction of the document being private, inside the service, of course.

Reply:
Definitely :+1: